flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -171,8 +171,8 @@ class SentryFlutterOptions extends SentryOptions {
   /// Only available on Android.
   bool enableNdkScopeSync = false;
 
-  /// Enable auto performance tracking by default.
-  bool enableAutoPerformanceTracking = true;
+  /// Enable auto performance tracing by default.
+  bool enableAutoPerformanceTracing = true;
```

Comment:
That's also part of https://github.com/getsentry/sentry-dart/issues/1177

Reply:
Ah ok, should we remove this here so we have a clear separation between the PRs?

Reply:
Yes, I guess this would trigger other breaking changes in general, unless this is a MUST DO in this PR and cannot be separated (for example, if merging this would break the current feature on Flutter/Cocoa).